google/cloud/bigtable/tests/admin_integration_test.cc

```diff
@@ -148,7 +150,10 @@ TEST_F(AdminIntegrationTest, DropAllRowsTest) {
        {column_family2, bigtable::GcRule::MaxNumVersions(10)},
        {column_family3, bigtable::GcRule::MaxNumVersions(10)}},
       {});
-  auto table = CreateTable(table_id, table_config);
+
+  ASSERT_STATUS_OK(table_admin_->CreateTable(table_id, table_config));
```

Comment:
I think this test could use `GetTable()`.

Reply:
Done.